programs/signals.py

```
@@ -0,0 +1,49 @@
+
+
+def _is_syncing() -> bool:
+    return getattr(_state, "syncing_navigators", False)
+
+
+def _set_syncing(value: bool) -> None:
+    setattr(_state, "syncing_navigators", value)
+
+
+@receiver(m2m_changed, sender=Program.navigators_sorted.through)
+def propagate_navigator_order_across_white_label(sender, instance: Program, action, reverse, model, pk_set, **kwargs):
```

Ruff (0.12.2)

Unused function argument: `sender`

(ARG001)